### DISCUSSION TITLE
[BUG] Error [ERR_REQUIRE_ESM]: require() of ES Module. dynamic import() which is available in all CommonJS modules.

### DISCUSSION THREAD

**Discussion by EvgeniyBudaev** [remix-run/remix#5408] [Q&A]
### What version of Remix are you using?

1.12.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

 I have created a simple project. https://github.com/EvgeniyBudaev/remix-ant/blob/main/app/routes/index.tsx

### Expected Behavior

The WordCloud component is displayed.

### Actual Behavior

I get an error:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module \remix-ant\node_modules\d3-interpolate\src\index.js from \remix-ant\node_modules\@antv\g-base\lib\animate\timeline.js not supported.
Instead change the require of index.js in \remix-ant\node_modules\@antv\g-base\lib\animate\timeline.js to a dynamic import() which is available in all CommonJS modules.
```

**Comment by barbinbrad** [comment]
Also get this bug. Should this be an issue? 

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/bbarbin/Code/carbon/node_modules/d3-scale/src/index.js from /Users/bbarbin/Code/carbon/node_modules/@visx/scale/lib/scales/band.js not supported.
Instead change the require of index.js in /Users/bbarbin/Code/carbon/node_modules/@visx/scale/lib/scales/band.js to a dynamic import() which is available in all CommonJS modules.
```

**Comment by barbinbrad** [comment]
For me, there was a problem with version 3 of these (visx/d3) packages. Downgrading to major version 2 solved it.

**Comment by EvgeniyBudaev** [comment]
@barbinbrad Can you show how you did it? I downgraded via overrides. We have a large commercial project and many different components are used, and manually overrides is not a good solution. This is problem. We cannot update a project that has been written for one year.

**Reply by barbinbrad** [reply]
Here's what my package.json looks like now: 

```
    "@visx/axis": "^2.18.0",
    "@visx/curve": "^2.17.0",
    "@visx/drag": "^2.17.0",
    "@visx/gradient": "2.17.0",
    "@visx/grid": "^2.18.0",
    "@visx/group": "^2.17.0",
    "@visx/pattern": "2.17.0",
    "@visx/responsive": "^2.17.0",
    "@visx/scale": "^2.18.0",
    "@visx/shape": "^2.18.0",
    "d3-array": "^3.2.2",
```
